README.md

```diff
@@ -18,14 +18,14 @@ https://github.com/Microsoft/ethr/releases/latest
 
 # Installation
 
-Note: go version 1.10 or higher is required building it from the source.
+Note: go version 1.11 or higher is required building it from the source.
 
 ## Building from Source
 
 ```
 git clone https://github.com/Microsoft/ethr.git
 cd ethr
-dep ensure -v
+go get ./...
```

Comment:
As this is a rather new feature in Go, I'd [point to the doc on how to use modules](https://github.com/golang/go/wiki/Modules#how-to-use-modules) and specify to set the `GO111MODULE=on` environment variable.

Reply:
done!